README.md

```diff
@@ -192,27 +161,25 @@ This architecture has several advantages:
 
 When a class includes `Hanami::Repository`, it will receive the following interface:
 
-  * `.persist(entity)` – Create or update an entity
-  * `.create(entity)`  – Create a record for the given entity
-  * `.update(entity)`  – Update the record corresponding to the given entity
-  * `.delete(entity)`  – Delete the record corresponding to the given entity
-  * `.all`   - Fetch all the entities from the collection
-  * `.find`  - Fetch an entity from the collection by its ID
-  * `.first` - Fetch the first entity from the collection
-  * `.last`  - Fetch the last entity from the collection
-  * `.clear` - Delete all the records from the collection
-  * `.query` - Fabricates a query object
+  * `#create(data)`     – Create a record for the given data (or entity)
+  * `#update(id, data)` – Update the record corresponding to the given id by setting the given data (or entity)
+  * `#delete(id)`       – Delete the record corresponding to the given id
+  * `#all`              - Fetch all the entities from the relation
+  * `#find`             - Fetch an entity from the relation by primary key
+  * `#first`            - Fetch the first entity from the relation
+  * `#last`             - Fetch the last entity from the relation
+  * `#clear`            - Delete all the records from the relation
 
-**A collection is a homogenous set of records.**
-It corresponds to a table for a SQL database or to a MongoDB collection.
+**A relation is a homogenous set of records.**
+It corresponds to a table for a SQL database or to a MongoDB relation.
```

Comment:
It looks like MongoDB refers to them as collections, so this line shouldn't change.